projects/management/commands/import_tock_projects.py

```diff
@@ -0,0 +1,81 @@
+import textwrap
+import logging
+import json
+from argparse import RawTextHelpFormatter
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils.text import slugify
+
+from projects.models import Project
+
+
+logger = logging.getLogger(__name__)
+
+
+class DryRunFinished(Exception):
+    pass
+
+
+class Command(BaseCommand):
+    help = textwrap.dedent("""
+    Imports projects exported from the Tock API.
+
+    To export projects from the Tock API, save the following URL
+    while logged-in to Tock from your browser:
+
+        https://tock.18f.gov/api/projects.json?page_size=1000
```

Comment:
Since Tock is behind MyUSA auth and it's a pain to make users manually retrieve their session cookie, it's easier to just tell the user to save the JSON export from their browser, I think. Alternatives welcome, though.